javascript/javascript1/week4/Highest-and-Lowest-Codewars-03.js

```diff
@@ -0,0 +1,9 @@
+function highAndLow(numbers) {
+  let numArray = numbers.split(" ");
+  let numList = numArray.map(Number);
+  let highest = Math.max(...numList);
+  let lowest = Math.min(...numList);
+  return highest + " " + lowest;
```

Comment:
From the previous comment we saw that `numList` is obsolete and can be skipped. Then we can reduce the lines of code, by removing `highest` and `lowest` as anyways they are used just a single time. Variables make most sense, when they can be reused in order to simplify code, but shouldn't be used to excess. I know it's a bit tricky to find the right balance, and often it can be just a matter of personal preference.

```suggestion
  return Math.max(...numList) + " " + Math.min(...numList);
```